tests/test_util.py

```diff
@@ -456,3 +459,46 @@ def test_df_parser(tmpdir):
     assert len(df.labels) == 1
     assert df.labels.get('label') == 'foobar barfoo'
 
+def test_df_parser_parent_env_arg(tmpdir):
+    tmpdir_path = str(tmpdir.realpath())
+    p_env = {
+        "test_env": "first"
+    }
+    df_content = """\
+FROM fedora
```

Comment:
Could use textwrap's `dedent()` here (as in other places) to make it easier to read.